Build on master fails with an empty m2 repository

On a machine that I haven't built syndesis from on (reproducable also via `mv ~/.m2 ~/.m2.backup`) the build fails:

```[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Syndesis ........................................... SUCCESS [  3.094 s]
[INFO] Common ............................................. SUCCESS [  0.025 s]
[INFO] Common :: Util ..................................... SUCCESS [  3.550 s]
[INFO] Common :: Model .................................... SUCCESS [  5.697 s]
[INFO] Extension .......................................... SUCCESS [  0.018 s]
[INFO] Extension :: API ................................... SUCCESS [  0.800 s]
[INFO] Extension :: Annotation Processor .................. SUCCESS [  0.947 s]
[INFO] Extension :: Archetype :: Spring Boot .............. SUCCESS [  1.453 s]
[INFO] Extension :: Archetype :: Java ..................... SUCCESS [  0.572 s]
[INFO] Extension :: Archetype :: Camel XML ................ SUCCESS [  0.496 s]
[INFO] Extension :: Bill of Materials (BOM) ............... SUCCESS [  0.067 s]
[INFO] Extension :: Converter ............................. SUCCESS [  0.934 s]
[INFO] Extension :: Maven Plugin .......................... SUCCESS [  2.087 s]
[INFO] Extension :: Example :: Log Step ................... SUCCESS [01:21 min]
[INFO] Integration ........................................ SUCCESS [  0.008 s]
[INFO] Integration :: API ................................. SUCCESS [  0.941 s]
[INFO] Integration :: Component Proxy ..................... SUCCESS [ 23.155 s]
[INFO] Integration :: Runtime ............................. SUCCESS [01:02 min]
[INFO] Integration :: Bill of Materials (BOM) ............. SUCCESS [  0.003 s]
[INFO] Integration :: Project Generator ................... SUCCESS [ 16.316 s]
[INFO] Connector .......................................... SUCCESS [  5.117 s]
[INFO] Connector :: Support :: Util ....................... SUCCESS [  0.677 s]
[INFO] Connector :: Support :: Verifier ................... SUCCESS [  0.896 s]
[INFO] Connector :: Support :: Maven Plugin ............... SUCCESS [ 19.960 s]
[INFO] Connector :: Twitter ............................... SUCCESS [  3.407 s]
[INFO] Connector :: FTP ................................... SUCCESS [  2.460 s]
[INFO] Connector :: SFTP .................................. SUCCESS [  0.849 s]
[INFO] Connector :: Dropbox ............................... SUCCESS [  5.328 s]
[INFO] Connector :: AWS S3 ................................ SUCCESS [02:29 min]
[INFO] Connector :: Support :: Test ....................... SUCCESS [  3.612 s]
[INFO] Connector :: ActiveMQ .............................. SUCCESS [ 21.731 s]
[INFO] Connector :: AMQP .................................. SUCCESS [ 33.888 s]
[INFO] Connector :: SQL ................................... SUCCESS [ 12.057 s]
[INFO] Connector :: MQTT .................................. SUCCESS [  3.986 s]
[INFO] Connector :: Support :: Processor .................. SUCCESS [  1.639 s]
[INFO] Connector :: HTTP .................................. SUCCESS [  5.017 s]
[INFO] Connector :: Salesforce ............................ SUCCESS [ 20.596 s]
[INFO] Connector :: Slack ................................. SUCCESS [  3.169 s]
[INFO] Connector :: Gmail ................................. SUCCESS [ 21.148 s]
[INFO] Connector :: Timer ................................. SUCCESS [  8.502 s]
[INFO] Connector :: ServiceNow ............................ FAILURE [  7.976 s]
[INFO] Connector :: Concur ................................ SKIPPED
[INFO] Connector :: Kafka ................................. SKIPPED
[INFO] Connector :: FHIR .................................. SKIPPED
[INFO] Connector :: Webhook ............................... SKIPPED
[INFO] Connector :: Log ................................... SKIPPED
[INFO] Connector :: Telegram .............................. SKIPPED
[INFO] Connector :: IRC ................................... SKIPPED
[INFO] Connector :: API Provider .......................... SKIPPED
[INFO] Connector :: Google Calendar ....................... SKIPPED
[INFO] Connector :: Google Sheets ......................... SKIPPED
[INFO] Connector :: Apache Kudu ........................... SKIPPED
[INFO] Connector :: OData ................................. SKIPPED
[INFO] Connector :: Support :: Catalog .................... SKIPPED
[INFO] Connector :: REST Swagger .......................... SKIPPED
[INFO] Server ............................................. SKIPPED
[INFO] Server :: DAO ...................................... SKIPPED
[INFO] Server :: JsonDB ................................... SKIPPED
[INFO] Server :: API Generator ............................ SKIPPED
[INFO] Server :: OpenShift Services ....................... SKIPPED
[INFO] Server :: Verifier ................................. SKIPPED
[INFO] Server :: Inspector ................................ SKIPPED
[INFO] Server :: Credential ............................... SKIPPED
[INFO] Server :: Endpoint ................................. SKIPPED
[INFO] Server :: Update Controller ........................ SKIPPED
[INFO] Server :: Controller ............................... SKIPPED
[INFO] Server :: Metrics :: JsonDB ........................ SKIPPED
[INFO] Server :: FileStore ................................ SKIPPED
[INFO] Server :: Logging :: JsonDB ........................ SKIPPED
[INFO] Server :: Monitoring ............................... SKIPPED
[INFO] Server :: Metrics :: Prometheus .................... SKIPPED
[INFO] Server :: Builder :: Maven Plugin .................. SKIPPED
[INFO] Server :: Runtime .................................. SKIPPED
[INFO] Server :: CLI ...................................... SKIPPED
[INFO] Server :: Builder :: Image Generator ............... SKIPPED
[INFO] S2I ................................................ SKIPPED
[INFO] Meta ............................................... SKIPPED
[INFO] UI ................................................. SKIPPED
[INFO] Documentation ...................................... SKIPPED
[INFO] Documentation :: Maven plugin ...................... SKIPPED
[INFO] Syndesis :: Docs ................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 08:57 min
[INFO] Finished at: 2019-01-23T09:23:06-05:00
[INFO] Final Memory: 318M/771M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project connector-servicenow: Could not resolve dependencies for project io.syndesis.connector:connector-servicenow:jar:1.6-SNAPSHOT: Failed to collect dependencies at org.apache.camel:camel-servicenow:jar:2.21.0.fuse-730029 -> org.apache.cxf:cxf-core:jar:3.2.7.fuse-730009: Failed to read artifact descriptor for org.apache.cxf:cxf-core:jar:3.2.7.fuse-730009: Could not transfer artifact org.apache.cxf:cxf-core:pom:3.2.7.fuse-730009 from/to jb-fuse-7-brew (http://download.eng.bos.redhat.com/brewroot/repos/jb-fuse-7-maven-build/latest/maven): download.eng.bos.redhat.com: Name or service not known: Unknown host download.eng.bos.redhat.com: Name orservice not known -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :connector-servicenow
ERROR: Last command exited with 1
```

Ideally the build shouldn't reference artifacts that aren't available externally.  Unless this is some transient error with the nexus server.  Currently worked around it by rsync'ing a repo from another machine that has all these artifacts already.

--- a/app/extension/bom/pom.xml
+++ b/app/extension/bom/pom.xml
@@ -28,7 +28,7 @@
 
   <properties>
     <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
-    <camel.version>2.21.0.fuse-730029</camel.version>
+    <camel.version>2.21.0.fuse-730033</camel.version>
   </properties>
 
     <!-- Metadata need to publish to central -->

--- a/app/integration/bom/pom.xml
+++ b/app/integration/bom/pom.xml
@@ -28,7 +28,7 @@
 
   <properties>
     <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
-    <camel.version>2.21.0.fuse-730029</camel.version>
+    <camel.version>2.21.0.fuse-730033</camel.version>
     <atlasmap.version>1.38.1</atlasmap.version>
   </properties>
 

--- a/app/pom.xml
+++ b/app/pom.xml
@@ -75,7 +75,7 @@
     <commons.compress.version>1.18</commons.compress.version>
 
     <!-- Global camel version used everywhere -->
-    <camel.version>2.21.0.fuse-730029</camel.version>
+    <camel.version>2.21.0.fuse-730033</camel.version>
 
     <dep.plugin.dependency.version>3.0.2</dep.plugin.dependency.version>
     <dep.plugin.pmd.version>3.10.0</dep.plugin.pmd.version>

